avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/FeatureIndicesConversion.scala

```diff
@@ -137,6 +154,29 @@ object FeatureIndicesConversion {
     }
   }
 
+  private def convertIdToSparseVector(
+    featureMapping: Map[String, Long],
+    discardUnknownEntries: Boolean,
+    filterZeros: Boolean): UserDefinedFunction = {
+    udf {
+      id: Long => {
+        Avro2TF.SparseVector(Seq(0), Seq(id.toFloat), filterZeros)
```

Comment:
Floats only have 24 bits of precision. Why are we converting longs to floats, rather than using vectors whose element type is long?

Reply:
This is not required. Please check new pull request (https://github.com/linkedin/Avro2TF/pull/62) I will close this pull request